### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by nhooyr**

### CODE REVIEW

File: runner.go [hunk 1]
```diff
@@ -152,6 +152,9 @@ func (r *runner) constructCommand(projectDir string) string {
 	// We start code-server such that extensions installed through the UI are placed in the host's extension dir.
 	cmd := fmt.Sprintf(`set -euxo pipefail || exit 1
 cd %v
+# This is necessary in case the .vscode directory wasn't created inside the container, as mounting to the host
+# extension dir will create it as root.
+sudo chown user:user ~/.vscode
```
Comment (nathanpotter) [inline runner.go]: Do we need to run `chown -R`?

Comment (nhooyr) [reply runner.go]: Mm we do.

Comment (nhooyr) [reply runner.go]: https://github.com/cdr/sail/commit/ca753921313c899a87d8cbda17b18c94cebf180c

Comment (nathanpotter) [reply runner.go]: Sorry about that, saw it after I approved

File: sail.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 (function() {
+    let oldonkeydown
```
Comment (scsmithr) [inline sail.js]: ol donkey down

File: sail.js [hunk 3]
```diff
@@ -8,6 +9,7 @@
     <div class="msg">Rebuilding container</div>
     </div>`
         // Prevent keypresses.
+        let oldonkeydown = document.body.onkeydown
```
Comment (scsmithr) [inline sail.js]: Should the `let` be omitted?

Comment (nhooyr) [reply sail.js]: Yea it should, nice catch.

**Pull request merged**